css/styles.css

```diff
@@ -0,0 +1,161 @@
+* {
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+body {
+  margin: 0 auto;
+  font-family: "Georgia", serif;
+  font-size: 18px;
+  padding: 0px;
+  width: 1024px;
+  background-image: url(images/coworking.jpg);
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-position: 103px 0px;
+}
+
+.header {
+  background-color: black;
+  height: 100px;
+  border: black 1px solid;
+  padding: 10px;
+}
+
+nav {
+  overflow: auto;
+  position: relative;
+  top: 10px;
+  left: 380px;
+}
+
+a {
+  color: white;
+  text-decoration: none;
+  font-family: arial;
+  padding-left: 10px;
+}
+
+.subtitle {
+  background-color: #eee;
+  border-top: yellow solid 3px;
+  height: 60px;
+  text-align: center;
+  padding-top: 15px;
+}
+
+.taste {
+  text-decoration: line-through;
+  color: darkgrey;
+}
+
+.are {
+ /* text-decoration: underline;
+  text-decoration-color: yellow;
+  text-underline-position: 3px;*/
+  border-bottom: 3px yellow solid;
+}
+
+.callout {
+  background-color:rgba(255,255,255, 0.8);
+  border: black 1px solid;
+  width: 25%;
+  height: 380px;
+  display: inline-block;
```

Comment:
since it's `position: absolute` you do not need the `display: inline` and the `float: right`